### PR TITLE
Fix AdminUser class conflict

### PR DESCRIPTION
**Pull request by benjamin-thomas**
In reference to issue #145

### CODE REVIEW

File: lib/godmin/generators/named_base.rb [hunk 1]
```diff
@@ -7,6 +7,14 @@ class NamedBase < Base
 
       private
 
+      def full_class_name
```
Comment (jensljungblad) [inline lib/godmin/generators/named_base.rb]: I think you could leverage the `class_path` method here:

```ruby
def full_class_name
  class_path.join("/").classify
end
```

Comment (benjamin-thomas) [inline lib/godmin/generators/named_base.rb]: Are you sure this is desirable?

Unless I'm mistaken, that code would get "expanded" into something similar to this

```ruby
namespace, name = Admin, 'AdminUser'
(namespace.name.split('::').map(&:underscore) + name.classify.deconstantize.split("::").map(&:undescore)).join('/').classify
```

That's quite a lot of transformation being chained here.

Comment (jensljungblad) [inline lib/godmin/generators/named_base.rb]: True, let's leave it as it is.

**Comment by jensljungblad** [comment]
Thanks!

**Pull request merged**